dodo.py

```diff
@@ -0,0 +1,7 @@
+def task_gen_parser():
+    """generate the standlone pidgy json parser"""
```

Comment:
You know i love `doit` and absolutely think it should be used for automating `importnb`'s docs, tests, linting, etc. but: It's probably worth exploring:
- not checking this in
- adding `lark` to the build system to enable editable installs from github
  - this will have little impact on folks installing from `.whl`, but

Reply:
yea you're right. the build and ci are kinda old. `importnb`s been neglected like a tomigochi. i'm gonna keep all this stuff in this pr cause it works then open another one fixing all the build stuff.